extensions/todoist/src/components/TaskActions.tsx

```diff
@@ -48,12 +62,32 @@ export default function TaskActions({
     }
   }
 
+  async function focusTask({ id, content }: Task) {
```

Comment:
You may want to add a `try/catch` block here as it's possible the user doesn't have the menu bar command enabled. 

<img width="726" alt="CleanShot 2022-12-21 at 14 01 13@2x" src="https://user-images.githubusercontent.com/16003285/208911414-42f214e1-308a-4b03-ba4e-7f1bbeea8b98.png">

The `type` should also be `UserInitiated` since `Background` is for scheduled intervals launches.